views/ABViewCore.js

```diff
@@ -839,10 +841,15 @@ module.exports = class ABViewCore extends ABMLClass {
                   allSaves.push(
                      // send a null for parent, so that the .save() wont trigger
                      // a save of the parent.
-                     v.copy(lookUpIds, null, options).then((copiedView) => {
```

Comment:
In the current design for the ```ABViewXXX``` classes, when a view ```.save()```s itself, it also tries to perform a ```.save()``` on it's parent if the view wasn't already included in the parent's ```._views```.

In the process of copying a view with, say 10 sub views.  The ```.parent``` view would perform 10 ```.saves()``` as each of it's copied views perform their ```.save()```.

According to the comments here, the ```null``` for the parent was supposed to prevent those additional `.saves()` while the view was copying itself and it's children.

I think the ```copiedView.parent = result;``` on line below was supposed to reconnect them and and then allow for saving later on.

So, can we keep the ```null``` and the ```copiedView.parent = result;```  below and still have everything work?

Reply:
Thank you for your reviewing.

If we don't have .parent value in the sub-page, then it will always create opsportal Nav view.
https://github.com/appdevdesigns/app_builder/blob/efbae63e5978982df428ffbf7aa01dba19f1237d/config/bootstrap.js#L894

Reply:
ok, that makes sense. 

In the new ```v2``` branch, we no longer have the Nav system, so don't be surprised if I put this optimization back in.

but for now, yes take this out.